tests/taglib/test_aiff.rs

```diff
@@ -0,0 +1,95 @@
+use crate::temp_file;
+use lofty::{Accessor, AudioFile, FileType};
+
+#[test]
+#[ignore]
+fn test_aiff_properties() {
+	let file = lofty::read_from_path("tests/taglib/data/empty.aiff", true).unwrap();
+
+	assert_eq!(file.file_type(), FileType::AIFF);
+
+	let properties = file.properties();
+	assert_eq!(properties.duration().as_secs(), 0);
+	assert_eq!(properties.duration().as_millis(), 67);
```

Comment:
```suggestion
	assert_eq!(properties.duration().as_millis(), 66);
```
Durations are normally +/- 1ms due to how TagLib rounds.

Reply:
I keep it for later, as some comment about this would be useful.